charts/grafana/templates/_helpers.tpl

```diff
@@ -139,7 +132,7 @@ Return the appropriate apiVersion for rbac.
 Return the appropriate apiVersion for ingress.
 */}}
 {{- define "grafana.ingress.apiVersion" -}}
-  {{- if and (.Capabilities.APIVersions.Has "networking.k8s.io/v1") (semverCompare ">= 1.19.x" (include "grafana.kubeVersion" .)) -}}
+  {{- if and (.Capabilities.APIVersions.Has "networking.k8s.io/v1") (semverCompare ">= 1.19-0" .Capabilities.KubeVersion.Version) -}}
```

Comment:
Where do I find an explanation for the `-0`?

Reply:
It's documented in the [Helm docs](https://helm.sh/docs/chart_template_guide/function_list/#semvercompare), but the short explanation is that `-0` is the lowest possible pre-release version. After I made the original changes in #505 I spent a bit of time looking around and this pattern seems to be idiomatic and once I figured out what it was doing I saw that it was the correct solution for where we want to constrain on a K8s release line.